### PR TITLE
Feature/address attributes for specific cryptocurrencies

### PR DESCRIPTION
**Pull request by skinderis**

### CODE REVIEW

File: src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js [hunk 1]
```diff
@@ -104,6 +115,18 @@ class ExchangeWidget extends Component {
     this.walletInputEl.focus();
   }
 
+  focusDestinationTag() {
```
Comment (BeOleg) [inline src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js]: never used

Comment (skinderis) [reply src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js]: without these dest tag, paym id or memo is not passed to http request header. maybe am missing something

File: public/locales/de/translations.json [hunk 2]
```diff
@@ -6,6 +6,9 @@
     "email": "Email",
     "address": "Adresse",
     "youraddress": "Geben Sie Ihre {{selectedCoin}} Adresse ein",
+    "destinationtag": "Destination tag",
```
Comment (BeOleg) [inline public/locales/de/translations.json]: @skinderis address seems to be missing

File: src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js [hunk 3]
```diff
@@ -47,6 +53,9 @@ class ExchangeWidget extends Component {
       }
 
       this.walletInputEl.focus();
+      this.destinationTagInputEl.focus();
+      this.paymentIdInputEl.focus();
+      this.memoInputEl.focus();
```
Comment (BeOleg) [inline src/components/Home/Hero/ExchangeWidget/ExchangeWidget.js]: Calling `focus` on several elements will place the cursor at the last element (field) the function was called on.

**Pull request merged**